recipes/nnetsauce/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.4" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 02955dd6e181462d5f363a95c7e3f46b474ba5377d9c785a886b1e590da26f04
```

Comment:
When I download the tar.gz and calculate the `sha256`, I'm getting something different:

```bash
$ wget https://github.com/Techtonique/nnetsauce/archive/0.8.4.tar.gz
$ openssl sha256 0.8.4.tar.gz
SHA256(0.8.4.tar.gz)= 3341147dff839728410703f59670858481a4520994d80fc1b2a821e9c52895da
```

This matches what the CI thinks it should be.

Reply:
@synapticarbors updated to match the sha256